src/Models/Link.php

```diff
@@ -68,6 +69,7 @@ public function getCMSFields(): FieldList
 
             $titleField = $fields->dataFieldByName('Title');
             $titleField->setTitle(_t('LinkField.LINK_FIELD_TITLE', 'Title'));
+            $titleField->setDescription(_t('LinkField.LINK_FIELD_TITLE_DESCRIPTION', 'If left blank, default title will be used'));
```

Comment:
```suggestion
            $titleField->setDescription(_t('LinkField.LINK_FIELD_TITLE_DESCRIPTION', 'If left blank, an appropriate default title will be used on the front-end'));
```

Just changing the wording a bit to more fully satisfy the following acceptance criterion:

> Feedback is provided to the user that they can leave LinkField title field blank.

By saying the default is "appropriate" is has more of a "you can leave this blank if you want to" feel to it.

Reply:
Updated